backend/src/main/java/dev/k2/validateXml/ValidateXmlService.java

```diff
@@ -3,18 +3,55 @@
 import dev.k2.commons.XsdFileRepository;
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
+import org.xml.sax.SAXException;
+import javax.xml.XMLConstants;
+import javax.xml.transform.Source;
+import javax.xml.transform.stream.StreamSource;
+import javax.xml.validation.Schema;
+import javax.xml.validation.SchemaFactory;
+import javax.xml.validation.Validator;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 
 @Service
 @RequiredArgsConstructor
 public class ValidateXmlService {
 
     private final XsdFileRepository xsdFileRepository;
 
-    public ValidationResponse validateXml(String schemaId, String xmlToValidate)
-    {
+    public ValidationResponse validateXml(String schemaId, String xmlToValidate) {
+
+
         byte[] xsd = xsdFileRepository.getFile(schemaId);
+        Source xmlSteam = new StreamSource(xmlToValidate);
```

Comment:
Constructor StreamSource(String) contains argument that represents URI to the xml, not content itself.